lib/common/resources/otel-collector-config.yml

```diff
@@ -44,6 +44,19 @@ spec:
             external_labels:
               cluster: "{{clusterName}}"
           scrape_configs:
+            {{ start kubecostJob }}
+            - job_name: kubecost
+              honor_labels: true
+              scrape_interval: 1m
+              scrape_timeout: 10s
+              metrics_path: /metrics
+              scheme: http
+              dns_sd_configs:
+              - names:
+                - dashboard.kubecost.avt.eks.aws.dev
```

Comment:
Why are we hardcoding this. We should pass this from `index.ts` via reading `cdk.json`

Reply:
Done